Bio-KBase-Auth/lib/Bio/KBase/AuthToken.pm

```diff
@@ -585,6 +592,54 @@ sub go_request {
 
 }
 
+sub _get_token {
+    my $self = shift @_;
+    my %p = @_;
+
+    my $json;
+    eval {
+	my $baseurl = $Bio::KBase::Auth::AuthSvcHost;
+#	my %headers;
+#        warn $p{'path'};
+	unless ($p{'path'}) {
+	    die "No path specified";
```

Comment:
I'm confused re baseurl vs path. Seems like it'd be simpler to just pass in the entire auth url like the py client does it.